### DISCUSSION TITLE
APP 2FA reject button placement leads to accidental rejection

### DISCUSSION THREAD

**Discussion by FluxZA** [community/community#15558] [Mobile]
When entering the number required for 2FA on the Github mobile app, the reject button starts out centered on the pop-up, exactly where the numeric input form appears after a second or two. This means that with a little muscle memory I end up tapping the middle of my phone screen as the pop-up appears, hitting the reject button accidentally. If rejecting the sign-in had a confirmation, or the button was simply moved, this issue would go away.

This is a minor grievance, but it has happened to me enough times to motivate the creation of this post.

I am using the Android version of the app on a Samsung Galaxy A32 if that is relevant.

Edit:
The exact cause of the issue seems to be that the position of the github pop-up shifts upwards after the keyboard loads onto the screen. The actual position of the reject button and the numeric input to not shift within the pop-up.

**Comment by stevepopovich** [comment]
Hey Rowan, thanks for the feedback and detailed description. I will pass this onto the team for prioritization!
